scripts/janitor-clusters.sh

```diff
@@ -12,9 +12,9 @@ do
   echo "  NAME: ${NAME}"
   echo "  CREATE_TIME: ${CREATE_TIME} / ${FORMATED_CREATE_TIME}"
 
-  YESTERDAY_DATE=$(date -I -d "yesterday")
-  if [[ ${FORMATED_CREATE_TIME} < ${YESTERDAY_DATE} ]]; then
-    echo "Cluster ${NAME}(${FORMATED_CREATE_TIME}) is older than ${YESTERDAY_DATE} will terminate"
+  CUTOFF_DATE=$(date -I -d "7 hours ago")
```

Comment:
how about 12 hours?

Reply:
I'm fine with anything. We can have them reaped after 7 hours for now, and once we have builds that actually successfully last longer than 7 hours we can run the janitor's cutoff accordingly.

Right now we have a pretty hard 6-hour cap, so keeping clusters around 6 hours after that seems wasteful, but I don't really feel strongly about it. 🤷

Reply:
7 is fine for me as well, just trying to balance the comments 😅